tools/plugins/com.liferay.ide.layouttpl.ui/META-INF/MANIFEST.MF

```diff
@@ -17,7 +17,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.wst.xml.core;bundle-version="[1.1.402,2)",
  org.eclipse.wst.common.frameworks.ui;bundle-version="[1.1.301,2)",
  org.eclipse.ui.editors,
- org.eclipse.wst.sse.ui;bundle-version="[1.1.100,2)",
+ org.eclipse.wst.sse.ui;bundle-version="[1.1.102,2)",
```

Comment:
why did you bump this to 102?  What version does that come from?

Reply:
from com.liferay.ide.portlet.ui/META-INF/MANIFEST.MF and com.liferay.ide.project.ui/META-INF/MANIFEST.MF